src/common/transformations/src/transformations/common_optimizations/common_optimizations.cpp

```diff
@@ -160,6 +161,7 @@ bool ngraph::pass::CommonOptimizations::run_on_model(const std::shared_ptr<ngrap
     decomp->add_matcher<ngraph::pass::SimplifyCTCGreedyDecoderSeqLen>();
     decomp->add_matcher<ngraph::pass::EinsumDecomposition>();
     decomp->add_matcher<ngraph::pass::SoftmaxDecomposition, false>();
+    decomp->add_matcher<ngraph::pass::SoftSignDecomposition>();
```

Comment:
As it is going to be applied in most cases,
please try to add SoftSign at least to CPU plugin activation layer tests:
https://github.com/openvinotoolkit/openvino/blob/04bbe98e58056e996ffb80b3ce97f0b87eb119b0/src/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/activation.cpp#L29

Reply:
Added.